contrib/pylibpod/pylibpod/libs/containers.py

```diff
@@ -0,0 +1,207 @@
+
+    def export(self, target):
+        """Export container from store to tarball.
+
+        TODO: should there be a compress option, like images?
+        """
+        with self._client() as podman:
+            results = podman.ExportContainer(self.id, target)
+        return results['tarfile']
+
+    def start(self):
+        """Start container, return id on success."""
+        with self._client() as podman:
+            results = podman.StartContainer(self.id)
+        return results['container']
+
+    def stop(self, timeout=25):
+        """Stop container, return id on success."""
+        with self._client() as podman:
+            results = podman.StopContainer(self.id, timeout)
+        return results['container']
+
+    def remove(self, force=False):
+        """Remove container, return id on success.
+
+        force=True, stop running container.
+        """
+        with self._client() as podman:
+            results = podman.RemoveContainer(self.id, force)
+        return results['container']
+
+    def restart(self, timeout=25):
+        """Restart container with timeout, return id on success."""
+        with self._client() as podman:
+            results = podman.RestartContainer(self.id, timeout)
+        return results['container']
+
+    def rename(self, target):
+        """Rename container, return id on success."""
+        with self._client() as podman:
+            # TODO: Need arguements
+            results = podman.RenameContainer()
+        # TODO: fixup objects cached information
+        return results['container']
+
+    def resize_tty(self, y, x):
+        """Resize container tty."""
+        with self._client() as podman:
+            # TODO: magic re: attach(), arguements
+            podman.ResizeContainerTty()
+
+    def pause(self):
+        """Pause container, return id on success."""
+        with self._client() as podman:
+            results = podman.PauseContainer(self.id)
+        return results['container']
+
+    def unpause(self):
+        """Unpause container, return id on success."""
+        with self._client() as podman:
+            results = podman.UnpauseContainer(self.id)
+        return results['container']
+
+    def update_container(self):
+        """Update container..., return id on success."""
+        with self._client() as podman:
+            results = podman.UpdateContainer(self.id)
+        self.refresh()
+        return results['container']
+
+    def wait(self):
+        """Wait for container to finish, return 'returncode'."""
+        with self._client() as podman:
+            results = podman.WaitContainer(self.id)
+        return int(results['exitcode'])
+
+    def stats(self):
+        """Retrieve resource stats from the container."""
+        with self._client() as podman:
+            results = podman.GetContainerStats(self.id)
+        obj = results['container']
+        return collections.namedtuple('StatDetail', obj.keys())(**obj)
+
+    def logs(self):
```

Comment:
This seems like something that is going to need arguments (e.g. whether or not to follow a stream, or whether to only get the last *n* lines).  But maybe those aren't supported by the varlink API yet.